src/main/java/net/dv8tion/jda/api/entities/MessageChannel.java

```diff
@@ -2261,8 +2446,6 @@ default MessagePaginationAction getIterableHistory()
      * @param  messageId
      *         The message to pin.
      *
-     * @throws IllegalArgumentException
-     *         if the provided messageId is not positive.
```

Comment:
Why was this removed?
The description "positive" is not correct, however, we definitely can still throw an exception if the provided id isn't a snowflake.

Reply:
I misunderstood the `isSnowflake` check. The throws documentation was added back for the case that `messageId` is not a valid snowflake.